### DISCUSSION TITLE
数据图表看板、自定义充值金额

### DISCUSSION THREAD

**Discussion by BennyThink** [BurnHair/docs#191] [网站公告]
近期，我们上线了数据图表看板功能。

您可以在[令牌页面](https://burn.hair/token) 查看今日和最近7日的消费分布与走势。
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/9c605eac-819a-445f-897b-1c9772ddb1a1">

在首页也可以通过点击切换按钮，显示数据图表看板或默认内容

---------

同时，为了方便[兑换码的购买](https://shop.burn.hair/)，我们提供了自定义购买金额的功能。您只需要输入或者拖动，即可自选。
<img width="1106" alt="image" src="https://github.com/user-attachments/assets/b36d13a2-0c17-49c4-8eae-488a3db5d7cf">

----------

如对新功能有建议，欢迎在本帖下方留言

**Comment by suikaremilia** [comment]
stripe没法付款，点付款按钮没反应

![Screenshot_2024-09-02_10-24-19](https://github.com/user-attachments/assets/38c91059-ca83-4d3c-9379-ed3b90f32c4d)

**Reply by BennyThink** [reply]
可能是 turnstile 没加载出来，你刷新一下页面再试试呢？

**Reply by suikaremilia** [reply]
换浏览器、切代理、隐私模式，都试了，加载不出来

**Reply by BennyThink** [reply]
按F12 打开控制台，有报错吗

**Reply by suikaremilia** [reply]
很离奇，F12开着再做一次就正常了

**Reply by BennyThink** [reply]
可能缓存的原因🤨

**Comment by xia0ne** [comment]
啥时候可以支持多条链的usdt呢 比如polygon :)

**Reply by BennyThink** [reply]
建议很好，但是目前现在不支持任何 USDT的充值，还是没时间没精力去做😭

**Comment by Souddt** [comment]
stripe付款界面无法加载，我代理挂全局也不管用
![image](https://github.com/user-attachments/assets/aba2352c-c3ec-48c3-9f51-12419cc518d3)

**Reply by BennyThink** [reply]
可以尝试直接在站内充值 https://burn.hair/token

**Reply by Souddt** [reply]
> 可以尝试直接在站内充值 https://burn.hair/token

好的
